kubernetes/table_kubernetes_ingress.go

```diff
@@ -85,13 +85,45 @@ func listK8sIngresses(ctx context.Context, d *plugin.QueryData, _ *plugin.Hydrat
 		return nil, err
 	}
 
-	ingresses, err := clientset.ExtensionsV1beta1().Ingresses("").List(ctx, metav1.ListOptions{})
-	if err != nil {
-		return nil, err
+	input := metav1.ListOptions{
```

Comment:
Can we use name or namespace as an optional key qual in the list function? It should work if I provide any one of them.